src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt

```diff
@@ -33,5 +35,31 @@ object DeeperCommandExecutor : IdofrontCommandExecutor() {
                     sender.info("${player.name} is in section ${section.key}")
             }
         }
+        "syncedtime" {
```

Comment:
Maybe we should start putting these commands under one "deeperworld" command? I think sectionon and secitonoff should probably be under it as well, maybe even do a `/deeperworld tp on/off` with an arg.

Reply:
Maybe make it a shorthand syntax instead? `/dw time set 1000` for example. Bit easier to use.

Reply:
Yeah you can have both with `("deeperworld" / "dw") { ... }`